docs/src/pages/docs/crocks/Async.md

```diff
@@ -1583,6 +1603,16 @@ Resolved([ 'cat', 'bat', 'imp' ])
   .chain(findLastValid)
   .fork(log('rej'), log('res'))
 //=> rej: "Nothing Found"
+
+Resolved(Last.empty())
+  .chain(lastToAsync('Left'))
+  .fork(log('rej'), log('res'))
+//=> rej: "Left"
+
+Resolved(Last('too know!'))
+  .chain(lastToAsync('Error'))
```

Comment:
Can we use `Left` her instead of `Error`, so we maintain the same logic btw the examples and it's easier to compare.

Reply:
Yeah good idea